[KOGITO-6998] Adding alias configuration option (#143)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc

A Kubernetes service endpoint can be used as a service URL if the target service is within the same cluster, such as `http://myservice.mynamespace.cluster.svc.local`.
====

> === Using URI alias

Reviewer: ```suggestion

```

> As an alternative to `kogito.sw.operationIdStrategy`, you can assign an alias name to an URI by using `workflow-uri-definitions` custom link:{spec_doc_url}#extensions[extension]. Then you can use that alias as configuration key and in function definitions. 

Reviewer: ```suggestion
As an alternative to `kogito.sw.operationIdStrategy`, you can assign an alias name to an URI by using `workflow-uri-definitions` custom link:{spec_doc_url}#extensions[extension]. Then you can use that alias as a configuration key in function definitions. 
```

Author: This change the meaning, I meant you can use the alias as configuration AND also use the alias in function definitions

Reviewer: Ah, okay. Done, I've updated the suggestion

> .Example Serverless Workflow

Reviewer: ```suggestion
.Example workflow
```

[source,json]
----
"extensions" : [ {
      "extensionid": "workflow-uri-definitions",
      "definitions": {
        "remoteCatalog": "https://my.remote.host/apicatalog/apis/123/document",
       }
     }
  ], 
  "functions": [
        {
          "name": "operation1",
          "operation": "remoteCatalog#operation1"
        }, 
        {
          "name": "operation2",
          "operation": "remoteCatalog#operation2"
        }
    ]
----

.Example property
[source, properties]
----
quarkus.rest-client.remoteCatalog.url=http://localhost:8282/
----

> In the previous example, Kogito uses `remoteCatalog` as configuration key.

Reviewer: ```suggestion
In the previous example, {product_name} uses `remoteCatalog` as configuration key.
```
